catalog/imaging/imaging-transformer-chipping/src/main/java/org/codice/alliance/imaging/chip/transformer/CatalogOutputAdapter.java

```diff
@@ -232,7 +232,7 @@ protected TemporaryFileBackedOutputStream createTemporaryFileBackedOutputStream(
     return new TemporaryFileBackedOutputStream();
   }
 
-  NitfSegmentsFlow getNitfSegmentsFlow(InputStream resourceInputStream)
+  private NitfSegmentsFlow getNitfSegmentsFlow(InputStream resourceInputStream)
```

Comment:
Had to make this private, because Camel now supports calling package-private methods (see https://issues.apache.org/jira/browse/CAMEL-13298), and it was getting confused between the two overloaded getNitfSegmentsFlow methods due to https://issues.apache.org/jira/browse/CAMEL-15810